jobs/awslogs/templates/bin/awslogs-scanner.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+INOTIFY_WAIT="/var/vcap/packages/inotify-tools/bin/inotifywait"
+
+mkdir -p /var/vcap/jobs/awslogs/conf.d
+
+export CONFIG_FILE=/var/vcap/jobs/awslogs/conf.d/all-vcap-logs.conf
+
+if [ ! -f ${CONFIG_FILE} ]; then
+  touch ${CONFIG_FILE}
+fi
+
+scan_for_logs() {
+  TMPCONF=$(mktemp)
+
+  for I in `find /var/vcap/sys/log -type f`
+  do
+    # deliberately excluding obviously non-ingestable files (only ASCII, text or empty files,
+    # no timestamp-rotated files) to reduce log readers and thus forestall lost log lines
+    echo $I | egrep -q 'log.[0-9][0-9][0-9]+$' && continue
+    file $I | egrep -q 'ASCII|text|empty' || continue
```

Comment:
I'm concerned this empty check might result in a race condition when a a log file is created (and thus inotify triggers) but not written to yet so we skip it here, but I couldn't make it happen in a bit of ad-hoc testing